Fix it by checking for `null` client before acting on it

 .../java/com/android/server/biometrics/BiometricServiceBase.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/services/core/java/com/android/server/biometrics/BiometricServiceBase.java b/services/core/java/com/android/server/biometrics/BiometricServiceBase.java
index 74079c19af2..6d90761ca2f 100644
--- a/services/core/java/com/android/server/biometrics/BiometricServiceBase.java
+++ b/services/core/java/com/android/server/biometrics/BiometricServiceBase.java
@@ -783,6 +783,7 @@ public abstract class BiometricServiceBase extends SystemService
     protected void handleEnumerate(BiometricAuthenticator.Identifier identifier, int remaining) {
         ClientMonitor client = getCurrentClient();
 
+        if(client == null) return;
         client.onEnumerationResult(identifier, remaining);
 
         // All templates in the HAL for this user were enumerated
-- 
2.23.0

